LEAF_Request_Portal/js/dialogController.js

```diff
@@ -151,9 +169,9 @@ dialogController.prototype.isComplete = function() {
 
 dialogController.prototype.setSaveHandler = function(funct) {
 	$('#' + this.btnSaveID).off();
-	var t = this;
+	let t = this;
     this.dialogControllerXhrEvent = $('#' + this.btnSaveID).on('click', function() {
-        if(t.isValid() == 1 && t.isComplete() == 1) {        	
+        if(t.isValid(true) == 1 && t.isComplete(true) == 1) {
```

Comment:
this syntax is weird.  from first glance it looks like this is setting isValid to true.  it's ok, just maybe in the future define a variable like "submitted = true/false" and then send to the isValid and isComplete functions.

Reply:
Yeah that would make more sense haha :D